packages/camera/camera_platform_interface/lib/src/events/camera_event.dart

```diff
@@ -235,3 +235,33 @@ class CameraErrorEvent extends CameraEvent {
   @override
   int get hashCode => super.hashCode ^ description.hashCode;
 }
+
+class CameraTimeLimitReachedEvent extends CameraEvent {
+  final XFile path;
+
```

Comment:
Should the event maybe also include the time limit that was reached? I have a feeling there might be a use case for this.

Reply:
I agree, this would be helpful information to have around.